code/game/gamemodes/miniantags/borer/borer.dm

```diff
@@ -392,10 +392,10 @@
 	content += "<table>"
 
 	for(var/datum in typesof(/datum/borer_chem))
-		var/datum/borer_chem/C = new datum()
-		var/datum/reagent/R = chemical_reagents_list[C.chemname]
-		if(C.chemname)
-			content += "<tr><td><a class='chem-select' href='?_src_=[UID()];src=[UID()];borer_use_chem=[C.chemname]'>[R.name] ([C.chemuse])</a><p>[C.chemdesc]</p></td></tr>"
+		var/datum/borer_chem/C = datum
+		var/datum/reagent/R = chemical_reagents_list[initial(C.chemname)]
+		if(initial(C.chemname))
```

Comment:
gotta have those dank performance memes, even if the proc is only initiated at the speed of the user